src/nvme/ioctl.h

```diff
@@ -3871,7 +3877,7 @@ struct nvme_io_args {
 	int fd;
 	__u32 timeout;
 	__u32 nsid;
-	__u32 reftag;
+	__u64 reftag;
```

Comment:
This type of change is not possible with the 1.0 release. This breaks our stable APi guarantee.

Reply:
Made changes to set the reftag field accordingly based on the version of the structure.